src/util/redisConnector.js

```diff
@@ -93,20 +109,18 @@ const RedisDB = {
       const bytes = Buffer.byteLength(valueToStore, "utf-8");
       await this.client.setex(key, dataExpiry, valueToStore);
       stats.gauge('redis_set_val_size', bytes, {
-        timestamp: Date.now()
       });
     } catch (e) {
-      stats.increment('redis_set_val_error', {
-        error: e,
-        timestamp: Date.now()
+      stats.redisError({
```

Comment:
This should be `stats.increment("redis_error", {...tags})` & we shouldn't define a specific stat in `stats.js`

Reply:
fixed